src/tui.rs

```diff
@@ -1944,16 +1979,14 @@ impl<'a> Context<'a, '_> {
 
     /// Creates a button with the given text.
     /// Returns true if the button was activated.
-    pub fn button(&mut self, classname: &'static str, text: &str) -> bool {
-        self.styled_label_begin(classname);
+    pub fn button(&mut self, classname: &'static str, text: &str, style: ButtonStyle) -> bool {
+        self.block_begin(classname);
```

Comment:
Instead of using a block wrapper for the button label, I think it'd be better to pass the `classname` to `button_label`, which can then pass it to `styled_label_begin`.